src/screens/Election/ElectionApplication.js

```diff
@@ -52,14 +52,14 @@ class ElectionApplication extends Component {
 		const positionsArray = _.orderBy(positions, iterateesPos, orderPos);
 
 		return (
-			<View style = { page }>
+			<SafeAreaView style = { page }>
 				{ this.renderNavBar() }
 				<View style = { contentStyle }>
 					{ this.showListPosition(positionsArray) }
```

Comment:
What's the difference between showListPosition and showApplyPosition. If possible try to name it in a way where that would make sense to someone who doesn't know the rest of the code very well.